admin/autoupdater/src/version.c

```diff
@@ -34,8 +34,10 @@ static int char_order(char c) {
 		return 0;
 	else if (isalpha(c))
 		return c;
-	else if (c == '~')
+	else if (c == 0)
```

Comment:
Everywhere else we use `'\0'` instead of `0` for comparison.
```suggestion
	else if (c == '\0')
```